docs/api/version-history.md

```diff
@@ -21,6 +21,9 @@ keywords: "API, Docker, rcli, REST, documentation"
   `DeviceInfo` objects, each providing details about a device discovered by a
   device driver.
   Currently only the CDI device driver is supported.
+* `DELETE /images/{name}` now supports a `platforms` query parameter. It accepts
+  an array of JSON-encoded OCI Platform objects, allowing for selecting a specific
+  platforms to delete content for.
```

Comment:
```suggestion
  an array of JSON-encoded OCI Platform objects, allowing for selecting a specific
  platform to delete content for.
```

Reply:
In that case, probably "to select the platforms for which to delete the content for" (or something along those lines)

Currently it say "a specific" (singular) followed by "platforms" plural 😅